bot/log.py

```diff
@@ -69,18 +35,11 @@ def setup() -> None:
         }
 
     if "COLOREDLOGS_LOG_FORMAT" not in os.environ:
-        coloredlogs.DEFAULT_LOG_FORMAT = format_string
+        coloredlogs.DEFAULT_LOG_FORMAT = core_logging.log_format._fmt
```

Comment:
This is the only thing I'm slightly sad about. Since I didn't surface the logging string used by bot core, this is the only way to get the raw logging string.

I think it's fine, but if anyone feels strongly about it I can update bot core to make this available in a variable.